app/controllers/api/v1/bookings_controller.rb

```diff
@@ -18,6 +17,6 @@ def create
 
   private
   def booking_params
-    params.permit(:name, :start_time , :id)
+    params.permit(:name, :start_time , :id, :building_id)
```

Comment:
`:building_id` should be `:building` Active Record will take care of this. That's my theory at least

Reply:
We tried but we don't make it. It dosen't work. maybe we need some help here.